rust-toolchain.toml

```diff
@@ -0,0 +1,5 @@
+[toolchain]
+channel = "1.62.0"
+components = [ "rustfmt", "clippy", "llvm-tools-preview" ]
+targets = [ "x86_64-unknown-linux-gnu", "x86_64-apple-darwin" ]
```

Comment:
I don't think we need this - folks will have the target that is appropriate for their machine, and we don't want to require the ability to cross compile.